its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -78,14 +78,6 @@ public static void verifyNoGuiWarnings(Orchestrator orchestrator, BuildResult bu
     assertThat(task.getWarningsList()).isEmpty();
   }
 
-  // Verify an AnalysisWarning is raised inside the SQ GUI (on the project dashboard)
-  public static void verifyGuiTestOnlyProjectAnalysisWarning(Orchestrator orchestrator, BuildResult buildResult, String language) {
```

Comment:
I believe this warning should still be kept. If a SQ project (i.e., MSBuild solution) has only TEST code, there probably is a problem with the configuration.

If we don't give this warning, users will be missing MAIN-only expected issues (e.g., vulnerabilities), and will think it's a problem in our analyzer.